src/main/java/echo/TaskList.java

```diff
@@ -0,0 +1,114 @@
+package echo;
+import java.io.FileNotFoundException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class TaskList {
+    List<Task> tasks;
+    public TaskList(List<Task> tasks) throws EchoException, FileNotFoundException {
+        this.tasks = tasks;
+
+    }
+
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+
+    public static String listToString(List<Task> tasks) {
+        StringBuilder listString = new StringBuilder();
+        int count = 1;
+        for (Task task : tasks) {
+            listString.append(task.getTypeLetter()).append("|").append(task.getIsDone()).append("|").append(task.getTaskDes()).append("|").append(task.getAdd()).append("\n");
```

Comment:
It might look nicer if the line is split into multiple lines

Reply:
Agreed, can consider having a line break